.github/workflows/ci.yml

```diff
@@ -5,52 +5,39 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
     - uses: actions/checkout@v1
-    - name: Install wget and configure Microsoft apt repository
-      run: |
-        sudo apt install -y wget apt-transport-https
-        wget -q https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
-        sudo dpkg -i packages-microsoft-prod.deb
-        sudo add-apt-repository universe
-        sudo apt update
-    - name: install .Net Core 2.1
-      run: |
-        sudo apt install dotnet-sdk-2.1
     - name: Nuke NSec and Secp256k1.Net from source tree to make sure we don't depend on them
       run: rm -rf src/NSec src/Secp256k1.Net
+    - name: Remove NET Core 3
+      run: sudo apt remove dotnet-{sdk,runtime,runtime-deps}-3.0
```

Comment:
if after removing .NETCore3, without installing .NETCore2.x, you can still run the tests, it means dotnetCore wasn't really uninstalled properly?

Reply:
you can have multiple versions and NETCore installed, and the images have what looks like pretty much all of them installed. So why do you think it should break?

Reply:
ah really? such a mess! (then make the name be `Remove .NETCore v3 (v2.x will be still available)`)